web_external/js/views/body/SessionView.js

```diff
@@ -93,40 +117,46 @@ minerva.views.SessionView = minerva.View.extend({
             this._disableSave();
         });
 
-        this.dataPanel = new minerva.views.DataPanel({
-            session: this.model,
-            collection: this.datasetsCollection,
-            parentView: this
-        });
-
-        this.mapPanel = new minerva.views.MapPanel({
-            session: this.model,
-            collection: this.datasetsCollection,
-            parentView: this
-        });
-
-        this.layersPanel = new minerva.views.LayersPanel({
-            collection: this.datasetsCollection,
-            parentView: this
-        });
-
-        this.jobsPanel = new minerva.views.JobsPanel({
-            parentView: this
-        });
-
-        this.analysisPanel = new minerva.views.AnalysisPanel({
-            parentView: this,
-            collection: this.analysisCollection,
-            datasetCollection: this.datasetCollection,
-            sourceCollection: this.sourceCollection
-        });
-
-        this.sourcePanel = new minerva.views.SourcePanel({
-            session: this.model,
-            sourceCollection: this.sourceCollection,
-            datasetCollection: this.datasetsCollection,
-            parentView: this
-        });
+        this.layout = {
```

Comment:
Is the layout the kind of thing that could go in the session model?  That would seem more versatile than storing a list of active panels and disabling those that aren't listed.  Then again, I've already demonstrated my lack of understanding of backbone.